beast/tools/tests/test_convert_hdf5_to_fits.py

```diff
@@ -0,0 +1,12 @@
+from astropy.tests.helper import remote_data
+from beast.tools import convert_hdf5_to_fits
+
+
+@remote_data
+def test_convert_hd5_to_fits():
+
+    # Pick some random .hd5 file to convert
+    data_fname = download_rename("beast_example_phat_seds_trim.grid.hd5")
+
+    # Convert the file
+    convert_hd5_to_fits.st_file(data_fname)
```

Comment:
It is better practice to import the function needed from `convert_hd4_to_fits` instead of the whole module.  In this case `from convert_hd5_to_fits import st_file` and then this line would be `st_file(data_fname).

Also, this test does not have an assert statement.  Is this intended?  Is this just a "smoke" test to make sure that the conversion happens?  It would be a more complete test if a cached version of the fits file was compared to the just created version to make sure the conversion happened correctly.